types/FrameTypes.ts

```diff
@@ -85,8 +74,7 @@ export type TextFrame = {
 
 export enum ImageSourceTypeEnum {
     url = 'url',
-    urlVariable = 'urlVariable',
-    connectorVariable = 'connectorVariable',
+    variable = 'variable',
     connector = 'assetProvider',
```

Comment:
this is changed because of the EditorAPI changes, `connector` is the right name, however, to avoid any breaking changes, we preferred to keep `assetProvider` value as well